modules/geo-layers/src/tile-3d-layer/get-frame-state.js

```diff
@@ -55,7 +54,7 @@ export function getFrameState(viewport, timeline) {
     },
     height,
     cullingVolume,
-    frameNumber: tick, // TODO: This can be the same between updates, what number is unique for between updates?
+    frameNumber: options.frameNumber || Date.now(), // TODO: This can be the same between updates, what number is unique for between updates?
```

Comment:
I think it is better to call `Date.now()` in the calling function in the layer with a comment that "we don't have anything better to offer", rather than build the hack into this function.